src/vanilla/DanModel/DanCodeGenerator.cs

```diff
@@ -2037,7 +2085,7 @@ public static JavaFile GetServiceClientJavaFile(CodeModelJv codeModel, Settings
                             }
                             else
                             {
-                                interfaceBlock.Annotation($"Headers({{ \"x-ms-logging-context: {codeModel.FullyQualifiedDomainName} {method.Name}\" }})");
+                                interfaceBlock.Annotation($"Headers({{ \"x-ms-logging-context: {GetFullyQualifiedDomainName(codeModel)} {method.Name}\" }})");
```

Comment:
You may know already-- x-ms-logging-context is dead because the callerMethod property on HttpRequest accomplishes the same thing for us. It's just that nobody has gotten around to removing it yet.

Reply:
Good point. I'll remove it.